src/Schema/MetaInformation.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GoetasWebservices\XML\XSDReader\Schema;
+
+/**
+ * Contains the information of additional attributes that are not part of the XSD specification:
+ * Formats like WSDLs can allow additional attributes to be added to the schemas.
+ * This class is used to store this information.
+ */
+class MetaInformation
+{
+    /**
+     * Links to the schema in which this information is contained.
+     * This context schema can be used to resolve an un-prefixed qname type.
+     *
+     * Example:
+     * wsdl:arrayType="int"
+     *
+     * value = "int"
+     * contextSchema = xsd : "http://www.w3.org/2001/XMLSchema"
+     *
+     * The type would be xsd:int
+     */
+    private Schema $contextSchema;
```

Comment:
this is not required, the extension attribute does not have to be referenced to another XSD schema.

this is perfectly valid:

```xml
    <complexType name="testType" xmlns:ns1="abc">
        <complexContent>
            <restriction base="xsd:string">
        <attribute ref="mytype" ns1:whatever="foo"/>
        </restriction>
    </complexContent>
    </complexType>
```

the namespace `abc` does not require to be in any imported schema

Reply:
It previsously added the schema of the surrounding context in order to resolve non-qualified type information. But I found another way to do it based. It is now removed - making the codechange easier again.